### PR TITLE
Fix file thumbnail cropping and rounded thumbnail corners

### LINKED ISSUE #5631: Thumbnails of images are cropped into squares
### Description

Thumbnails of images are cropped into squares
image

### Steps To Reproduce

1. Go to Pictures folder
2. See image thumbnails being cropped into squares

### Expected behavior

Thumbnails of images to not be cropped

### Files Version

1.9.0.0

### Windows Version

Windows 10 21H1 19043.1083

### Relevant Assets

_No response_

### LINKED ISSUE #5682: Grid View should have Rounded Corners
Honestly, I think grid view would look better with rounded corners. In the current setup, the video and image thumbnails have straight corners, which don't really match with the design.
image

### PR DESCRIPTION
**Pull request by winston-de**
**Resolved / Related Issues**
Items resolved / related issues by this PR.
- Closes #5682
- Closes #5631 

**Details of Changes**
Add details of changes here.
- Switch to single item thumbnail mode and change resize mode
- Round corners for grid view thumbnails
- Show proper file preview images in preview pane and grid view
- Fix issue where file name would sometimes be cut off in tiles view

**Validation**
How did you test these changes?
- [x] Built and ran the app
- [ ] Tested the changes for accessibility

**Screenshots (optional)**
image

### CODE REVIEW

**Comment by yaichenbaum** [comment]
@winston-de can you resolve the merge conflict?

**Comment by yaichenbaum** [comment]
@winston-de should the thumbnails in the details layout have rounded corners as well?

**Comment by winston-de** [comment]
I'm a bit conflicted about that, it almost feels like too much rounding even with just 1px.
image

File: Files/Views/LayoutModes/DetailsLayoutBrowser.xaml [hunk 1]
```diff
@@ -497,16 +497,20 @@
                                     Opacity="{x:Bind Opacity, Mode=OneWay}"
                                     TabFocusNavigation="Local"
                                     Tag="ItemImage">
-                                    <Image
-                                        x:Name="Picture"
+                                    <ContentPresenter
```
Comment (yaichenbaum) [inline Files/Views/LayoutModes/DetailsLayoutBrowser.xaml:500]: Can you make this change to the column layout as well?

**Review by yaichenbaum — APPROVED** [review]
LGTM

File: Files/ViewModels/ItemViewModel.cs [hunk 2]
```diff
@@ -729,7 +729,9 @@ private async Task LoadItemThumbnail(ListedItem item, uint thumbnailSize = 20, I
                         var matchingStorageFile = (StorageFile)matchingStorageItem ?? await GetFileFromPathAsync(item.ItemPath);
                         if (matchingStorageFile != null)
                         {
-                            using var Thumbnail = await matchingStorageFile.GetThumbnailAsync(ThumbnailMode.ListView, thumbnailSize, ThumbnailOptions.ResizeThumbnail);
+                            var mode = thumbnailSize < 80 ? ThumbnailMode.ListView : ThumbnailMode.SingleItem;
+
+                            using var Thumbnail = await matchingStorageFile.GetThumbnailAsync(mode, thumbnailSize, ThumbnailOptions.UseCurrentScale);
```
Comment (gave92) [inline Files/ViewModels/ItemViewModel.cs:734]: Should this be done also on line 805 (when loading a folder thumbnail)?

Comment (duke7553) [reply Files/ViewModels/ItemViewModel.cs:734]: @gave92 I also wonder whether my addition of `ThumbnailOptions.ReturnOnlyIfCached` on that line has something to do with it?

Comment (winston-de) [reply Files/ViewModels/ItemViewModel.cs:734]: oddly, this seems to break thumbnails randomly
image

**Comment by yaichenbaum** [comment]
@winston-de does this fix still work?

**Comment by gave92** [comment]
We reverted it in #5842

**Comment by gave92** [comment]
Opened issue #9129 to keep track of this.

**Comment by gave92** [comment]
Nope the breaking change for this was in #7881.

**Pull request merged**